src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/AutoInsert/AutoClosingTagOnAutoInsertProvider.cs

```diff
@@ -18,7 +18,7 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.AutoInsert
     internal class AutoClosingTagOnAutoInsertProvider : RazorOnAutoInsertProvider
     {
         // From http://dev.w3.org/html5/spec/Overview.html#elements-0
-        public static readonly HashSet<string> VoidElements = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+        public static readonly HashSet<string> VoidElements = new HashSet<string>(StringComparer.Ordinal)
```

Comment:
This doesn't feel like the right fix to me. If I don't have a tag helper called "Input" then we _should_ be self closing it, right, since its a html element? ditto if someone types `<BR>` because they love shouting, etc.

Seems like the fix should be more specific to "if the element matches a known tag helper then don't self-close".

Reply:
Ya you're right. I took the easy route 😆. 

Working on a changeset to make this more resilient

Reply:
🆙 📅 d